Notice.md

```diff
@@ -6,50 +6,99 @@ the open-source TEE maintained by Linaro, with initial contributions from
+to be compatible with any isolation technology suitable for the TEE concept and
+goals, such as running as a virtual machine or on a dedicated CPU.
 
 The main design goals for OP-TEE are:
--	Isolation - the TEE provides isolation from the Rich OS (typically,
-	Linux/Android) and it protects the Trusted Applications (TAs) it
-	executes from each other, using underlying HW support,
--	Small footprint - the TEE should remain small enough so that the TEE
-	core, including all the code and data required to provide isolation, can
-	reside in a reasonable amount of on-chip memory,
--	Portability - the TEE must be easily pluggable to different
-	architectures and available HW, and it has to support various setups
-	such as multiple TEEs or multiple client OSes.
+-	Isolation - the TEE provides isolation from the non secure OS and
+	protects the loaded Trusted Applications (TAs) from each other using
+	underlying HW support,
+-	Small footprint - the TEE should remain small enough to
+	reside in a reasonable amount of on-chip memory as found on ARM
+	based systems,
+-	Portability - the TEE aims at being easily pluggable to different
+	architectures and available HW and has to support various setups
+	such multiple client OSes or multiple TEEs.
+
 
 Repository structure
 ------
 
-OP-TEE is composed of three gits:
--	The optee-client git, containing the source code for the TEE client
-	library in Linux. This component provides the TEE Client API as defined
-	by the <a href="https://www.globalplatform.org/specificationsdevice.asp">GlobalPlatform
-	TEE standard</a>. It is distributed under the BSD 2-clause open-source license.
--	The optee_os git, containing the source code for the TEE OS itself. This
-	component provides the TEE Internal APIs as defined by the
-	GlobalPlatform  TEE standard to the Trusted Applications that it
-	executes. It is distributed mostly under the BSD 2-clause open-source
-	license. It includes few external files under BSD 3-clause license or
-	other free software licenses.
--	The optee_linuxdriver git, containing the source code for the TEE driver
-	in Linux. This component implements a generic TEE driver, designed
-	primarily for TEE implementations that rely on the ARM
-	TrustZone(R)technology. It is distributed under the GPLv2 open-source
-	license. Please note that re-distribution under other versions of the
-	GPL license is not allowed. The rationale behind this limitation is to
-	ensure that this code may be used on products which have security
-	devices which prevent reloading the code. Such security devices would be
-	incompatible with some licenses such as GPLv3 and so distribution under
-	those licenses would be inconsistent with this goal. Therefore it is
-	recommended that care be taken before redistributing any of the
-	components under other license terms than those provided here.
+OP-TEE comes with several components:
+-	a secure privileged layer, executing at ARM secure PL-1 level,
```

Comment:
Should we use EL-1 instead?

Reply:
PL-1 => priv land, PL-0 unpriviledge land: applies to both ARMv7-A and ARMv8-A (where PL-0/1/2 = EL-0/1/2):
http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.den0024a/ch03s02.html

Reply:
update to lastest ref:
https://developer.arm.com/products/architecture/a-profile/docs/100878/latest/changing-exception-levels